README.md

```diff
@@ -311,4 +311,24 @@ Firestarter implements support for [ACF Options](https://www.advancedcustomfield
 firestarter()->settings()->get('site_logo')
 ```
 
-TG: We should add here more information about this functionality, for me, it wasn't clear why I should use it instead of the "classic" get_field function.
+### WEBP
+
+If you use the ACF Pro plugin you can enable the WebP functionality in the dashboard, just go to the "Firestarter -> Media Settings".
+To make it works you need to update your `.htaccess` files with the following rules:
+
+```
+# CACHING | WebP Support | https://do.co/3aIKIb8
+<IfModule mod_rewrite.c>
+	RewriteCond %{HTTP_ACCEPT} image/webp
+	RewriteCond %{REQUEST_URI}  (?i)(.*)(\.jpe?g|\.png)$
+	RewriteCond %{DOCUMENT_ROOT}%1%2.webp -f
+	RewriteRule (?i)(.*)(\.jpe?g|\.png)$ %1%2\.webp [L,T=image/webp]
+
+	RewriteCond %{HTTP_ACCEPT} image/webp
+	RewriteCond %{REQUEST_URI}  (?i)(.*)(\.jpe?g|\.png)$
+	RewriteCond %{DOCUMENT_ROOT}%1.webp -f
+	RewriteRule (?i)(.*)(\.jpe?g|\.png)$ %1\.webp [L,T=image/webp]
+</IfModule>
+```
+
+If you don't use the ACF Pro plugin and you would like to enable this functionality then you can go to the `app -> Media -> WEBP.php` file and set `$this->convertImages` to `true`.
```

Comment:
Add info on how to confirm it is working, add info that this is intended for apache servers